components/AlertNotification/AlertNotification.test.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { render } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
```

Comment:
Did the build not fail for having an unused `screen`?

Reply:
No I don't think so.